src/thank-you/contact.njk

```diff
@@ -3,7 +3,5 @@ title: Thank you for contacting us
 subtitle: Schedule a call now
 description: 
 downloadFollowUp: false
-hubspot:
-    script: "hubspot/hs-book-meeting.njk"
-    dataSrc: https://meetings-eu1.hubspot.com/michael-davis?embed=true
```

Comment:
Hi @Hasmin-AC removing this won't do the trick since the template is expecting this info. We need to change the template.

I'm opening an issue for this one, we also need to decide how the new layout it's going to be.

Reply:
I see. So yes we'll need a new template. How about under the thank you messages, we have a block for the latest on the. blog and the Cloud sign-up CTA? Below that will be the logos...